### PR TITLE
Load Singleplayer DMX values

### PR DESCRIPTION
**Pull request by ampreeT**
This forces the game to reload the weapon parameters on level initialization and to use singleplayer values in a cooperative map.

Fixes some issues in #144.

**Notable Changes:**

- Crowbar's attack speed is now same as singleplayer
- MP5 clip size now uses singleplayer values (`45` to `30`)
- Weapon damage values now matches singleplayer values

### CODE REVIEW

**Review by Alienmario — APPROVED** [review]
LGTM. Are there any prediction issues?

File: scripting/include/srccoop/blackmesa/entitypatch.inc [hunk 1]
```diff
@@ -1120,3 +1120,15 @@ public MRESReturn Hook_Dissolve(int _this, DHookReturn hReturn, DHookParam hPara
...
 }
+
+// Switches between using singleplayer and multiplayer weapon values.
+// This will get triggered by the command `params_reload_server`.
+public MRESReturn Hook_CParamsManager_InitInstances(int _this, Handle hReturn, Handle hParams)
+{
+	// TODO:
+	// Currently, this is getting only called during level init, but
+	// we also need to call this if the feature is turned on/off mid-round.
+	CParamsManager pParamsManager = CParamsManager(view_as<Address>(_this));
```
Comment (Alienmario) [inline scripting/include/srccoop/blackmesa/entitypatch.inc:1131]: If the callback is defined with Address type, no need to cast

Comment (ampreeT) [reply scripting/include/srccoop/blackmesa/entitypatch.inc]: Fixed.

**Comment by ampreeT** [comment]
> LGTM. Are there any prediction issues?

The only prediction issue that I have found is holding the reload button while the MP5's clip size is full at `30`. The client still thinks that the MP5 clip size is still `45`.

**Pull request merged**